src/main/java/org/kuali/kfs/module/ld/document/SalaryExpenseTransferDocument.java

```diff
@@ -145,21 +146,21 @@ public boolean answerSplitNodeQuestion(String nodeName) throws UnsupportedOperat
      * @return boolean
      */
     protected boolean checkOjbectCodeForWorkstudy() {
```

Comment:
This method is misspelled in base code, like it is here. We may want to report this to KualiCo if they haven't fixed it yet.

Reply:
added to sub-task to report to KualiCo